pkg/reconciler/kubernetes/tektonhub/tektonhub.go

```diff
@@ -175,12 +175,12 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, th *v1alpha1.TektonHub)
 
 	th.SetDefaults(ctx)
 
-	// reconcile target namespace
-	if err := common.ReconcileTargetNamespace(ctx, nil, th, r.kubeClientSet); err != nil {
-		logger.Errorw("error on reconciling targetNamespace",
-			"targetNamespace", th.Spec.GetTargetNamespace(),
-			err,
-		)
+	tc, err := r.operatorClientSet.OperatorV1alpha1().TektonConfigs().Get(ctx, v1alpha1.ConfigResourceName, metav1.GetOptions{})
+	if err != nil {
```

Comment:
@pratap0007 still it is possible to install Hub without `TektonConfig` right?
if yes, ignore resource not found error.

Reply:
Good catch, I think with this would not work if TektonConfig is not there